### PR TITLE
implement forge links to axiom

### PR DESCRIPTION
**Pull request by geekflyer**
### <samp>🤖 Generated by Copilot at 92066f3</samp>

This pull request adds support for Axiom logs to the forge test script and its test suite. It adds new functions to generate Axiom links for the test runner and node pods, and modifies the existing functions to include them in the comments. It also adds new fixtures and updates some function names and test cases to improve clarity and consistency.

### CODE REVIEW

File: testsuite/forge.py [hunk 1]
```diff
@@ -488,6 +488,58 @@ def get_humio_logs_link(
...
 
+def get_axiom_link_for_test_runner_logs(
+    forge_namespace: str,
+    time_filter: Union[bool, Tuple[datetime, datetime]],
+) -> str:
+    """Get a link to the forge test runner logs in axiom for a given test run in a given namespace"""
+
+    apl_query = f"""
+        k8s
+        | where ['k8s.cluster'] contains "forge" and ['k8s.container_name'] != "calico-node" and ['k8s.namespace'] != "calico-apiserver" and ['k8s.container_name'] != "kube-proxy" and
```
Comment (perryjrandall) [inline testsuite/forge.py:499]: can we break this up so its a little more readable?

```
constraints = [
  'cluster contains forge',
  'container_name != calico',
  'namespace != calico',
  ...
]
'k8s | where ' + ' and '.join(constraints)
```

Or can we keep the saved query server side?

File: testsuite/forge.py [hunk 2]
```diff
@@ -488,6 +488,58 @@ def get_humio_logs_link(
...
+    return logs_url
+
+
+def get_axiom_link_for_node_logs(
+    forge_namespace: str,
+    time_filter: Union[bool, Tuple[datetime, datetime]],
+) -> str:
+    """Get a link to the node logs in axiom for a given test run in a given namespace"""
+
+    apl_query = f"""
```
Comment (perryjrandall) [inline testsuite/forge.py:514]: ditto

**Review by perryjrandall — APPROVED** [review]

**Comment by perryjrandall** [comment]
sheepit

File: testsuite/forge.py (hunk 1 above)
Comment (geekflyer) [reply testsuite/forge.py:499]: we can't keep it server side yet (they're working on it though).
In terms of breaking this up - I don't think this a good idea. If anything breaking this up makes it less readible as it's less-copy and pasteable into axiom's UI and vice versa.

**Pull request merged**